browser-test/src/admin_predicates.test.ts

```diff
@@ -314,10 +317,62 @@ describe('create and edit predicates', () => {
       await logout(page)
     })
 
+    it('multiple values and multiple questions', async () => {
```

Comment:
We need to do this same test for eligibility too since from the user perspective they are different actions.

Reply:
I'm concerned about the browser tests taking a long time to run when we have so much duplication in the test logic

Reply:
I hear ya, but the foremost point of this testing is correctness not expediency.

These UIs are different sets of visual controls to the user that affect different system behavior, and as such they should have their own testing.

The fact that they are backed by mostly the same code isn't something these tests should consider.  As well soon I'll restrict which questions can be in Eligibility conditions so that will start to effectively diverge the User experience